### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by pullingrm12**
Basic Home Page

### CODE REVIEW

File: frontend/src/Header.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import styled from 'styled-components';
+
+const Searchdiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: right;
+  padding: 10px;
+`
+
+function GetHeader(){
```
Comment (jlaksana) [inline frontend/src/Header.js:12]: we usually name components just what they are. This would just be Header

File: frontend/src/Header.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            <div className="dropdown">
+              <button className="dropbtn">Sign In</button>
+            </div>
+          </h2>
+        </nav>
+      </header> 
+    );
+}
+
+function GetPun() {
```
Comment (jlaksana) [inline frontend/src/Header.js:40]: Same here. Name is just Pun. Probably better to make a separate file for it too

File: frontend/src/Header.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    function GetAnswer() {
+        return (
+            pun_a[randomNum]
+        );
+    }
+
+    return (
+        <div className="App-body">
+            <h2 className="App-h2">&#127820; Pun-ana of the Day : &#127820;</h2>
+            &#x1F34C;<GetQuestion />
```
Comment (jlaksana) [inline frontend/src/Header.js:60]: you can inject javascript in html by using {}
can do:
{pun_q[randomNum]}

File: frontend/src/Header.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            <br></br>
+            <br></br>
+            &#x1F34C;<GetAnswer />
+        </div>
+    );
+}
+
+function Header() {
```
Comment (jlaksana) [inline frontend/src/Header.js:68]: I think this should be Home instead. probably better to make a new file for the Home component

File: frontend/src/App.css [hunk 5]
```diff
@@ -36,3 +63,54 @@
...
+}
+
+.dropdown:hover .dropcont {
+  text-align: left;
+  padding-left: 10px;
+  display: block;
+}
+
+.dropdown:hover .dropbtn {
+  background-color: #E55B7E;
```
Comment (jlaksana) [inline frontend/src/App.css:115]: i think we can add a shadow on the button to make it a bit spicier

File: frontend/src/Header.js (hunk 4 above)
Comment (pullingrm12) [reply frontend/src/Header.js:68]: So have one file for Header, Pun, and Home and then just have Home call the function in Header and Pun?

Comment (jlaksana) [reply frontend/src/Header.js:68]: Yes! Also feel free to make a components folder to put all of them in

Comment (pullingrm12) [reply frontend/src/Header.js]: Ok I pushed again, let me know if that is good

**Review by jlaksana — APPROVED** [review]
Looks great! Amazing job

**Pull request merged**